src/printer.js

```diff
@@ -1953,7 +1953,7 @@ function printMemberChain(node, options, print) {
   if (hasMultipleLookups) {
     const currPrinted = print(FastPath.from(curr));
     const nodesPrinted = nodes.map(node => ({
-      property: print(FastPath.from(node.property)),
+      property: printMemberLookup(FastPath.from(node), print),
```

Comment:
`node` is not a good variable name here (my fault). It's not actually supposed to be a valid AST node. It just is a collection of AST nodes that I need to print individually. The fact that this works is a coincidence; for example `node` here has a `call` property which is not a valid AST property.

What you need to do is on line 1929, instead of adding `property: curr.callee.property` to the object, change it to something like this: `member: curr.callee`. Keep the MemberExpression around, not just the property. And then in the code here you can do `printMemberLookup(FastPath.from(node.member), print)`.